.github/workflows/bioconda-import.yaml

```diff
@@ -25,22 +22,29 @@ jobs:
         GITHUB_USER: ${{ secrets.GITHUB_USER }}
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
+        # since we're switching branches, preserve original git-utils scripts
+        mv ${{ github.workspace }}/content/scripts/git-utils ${{ github.workspace }}/git-utils
```

Comment:
Good thinking! However, I think we should follow @bgruening 's suggestion to externalize all ecosystem utilities to an external repository. (not now though, just in the long run)

Reply:
that's a good idea, we can make scripts folder as a separate repository and put here a submodule (or not, CI won't use a submodule anyway). 

Then we would have a repo with just pure "data". I would do it right away, but I don't have rights to create a repo here

Reply:
I will make sure you get the rights to do so, it would be a shame if you were to be stopped by that kind of reason ;)